website/src/libs/api-v3/firebase.js

```diff
@@ -0,0 +1,68 @@
+import Firebase from 'firebase';
+import nconf from 'nconf';
+const IS_PROD = nconf.get('IS_PROD');
+const FIREBASE_CONFIG = nconf.get('FIREBASE');
+const FIREBASE_ENABLED = IS_PROD && FIREBASE_CONFIG.ENABLED === 'true';
```

Comment:
Do we want to make this a prod only thing? Can devs sign up for test accounts with Firebase?